docs/self-hosted-appcircle/installation-podman.md

```diff
@@ -0,0 +1,990 @@
+
+```bash
+./ac-self-hosted.sh -n "spacetech" up
+```
+
+You should see
+
+> "Podman login cred not found. Trying to login now..."
+
+in command output and then
+
+> "Login Succeeded"
+
+which shows us successful artifact registry login.
+
+If you get any error for some reason at this step, you can remove `${XDG_RUNTIME_DIR}/containers/auth.json` file to reset and execute same command again to retake same steps.
+
+:::
+
+:::info
+
+At first run, it will pull once all container images needed from artifact registry. Also there are some migration steps taken for once which prepares system for other operations.
+
+So it may need up to ~20 min to system be up according to your internet connection speed and CPU power. But recurrent boots will take a couple of minutes and will have shorter durations.
+
+:::
+
+Now you can check system health and gain an overview of the status.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" check
+```
+
+It will give a quick health summary. You should see below message on success.
+
+```
+ All services are running successfully. Project name is spacetech
+```
+
+If you want to get details about podman services, you may run the following commands.
+
+```bash
+cd projects/spacetech/export
+podman-compose ps
+```
+
+If everything is okay, then you should see service statuses as "running", "running (healthy)" or "exited (0)".
+
+![](https://cdn.appcircle.io/docs/assets/be-962-docker-compose-ps.png)
+
+:::caution
+
+#### Vault
+
+All secret data, including the API keys, signing identities, environment variables, and secrets are stored in an HashiCorp Vault. OAuth tokens and SSH keys, used in build pipeline, are also stored securely in HashiCorp Vault.
+
+Vault is a tool for securely accessing secrets. It's an important and required service for whole self-hosted Appcircle server. If it's status is `unhealthy`, secrets will be inaccessible and most CI/CD functions won't work properly.
+
+For this reason, **before starting to use self-hosted Appcircle server** within your organization, make sure you check **vault service** status in container list above. Its **status must be `healthy`**.
+
+While you're working on configuration back and forth, it's status may become `unhealthy` in some way.
+
+In this case, stop all services with data cleanup.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" reset
+```
+
+Then make a new export and start services. Refer to [reset configuration](./installation-podman.md#reset-configuration) section for more details.
+
+:::
+
+:::info
+
+Self-hosted Appcircle server uses some ports for communication.
+
+Below ports must be unused on system and dedicated to only Appcircle server usage.
+
+
+- `80`
+- `443`
+- `8080`
+- `8443`
+
+Appcircle server will listen on `8080` and `8443` ports by default for HTTP and HTTPS connections.
+You can get a list of up-to-date ports used by podman with below command.
+
+```bash
+sudo netstat -tulpn | grep LISTEN | grep podman
```

Comment:
Is this line correct for Podman case? At Centos Stream 8 :point_down: 

```bash
[centos@centosstream8 export]$ sudo netstat -tulpn | grep LISTEN | grep podman
[centos@centosstream8 export]$ netstat -tulpn | grep LISTEN | grep podman
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
[centos@centosstream8 export]$
```

Reply:
You are right, that is not correct, process name of podman looks like 'rootlessport'
<img width="1448" alt="image" src="https://github.com/appcircleio/appcircle-docusaurus/assets/58513283/3a67662d-1f64-4096-96df-7f2b35806a1f">

Reply:
Fixed now.